README.md

```diff
@@ -10,3 +10,14 @@ See [EasterEggDetectorSystem](src/main/java/org/terasology/blockdetector/easter/
 ## Credits
 
 [ScannerBeep.ogg](assets/sounds/ScannerBeep.ogg) by kalisemorrison @ [freesound](https://www.freesound.org/people/kalisemorrison/sounds/202530/).
+
+
+# Testing
+
+This module also have tests for the BlockDetectorSystemImpl class. You can find the code [here](src/test/java/org/terasology/blockdetector/systems)
+To run the test, just simply run the code from the IDE :)
+Unit Tests available for these methods :
```

Comment:
`Unit Tests available` -> `Unit Tests are available`
 `for these methods :` -> `for these methods:` (remove the space at the end)